cardano-cli/test/cardano-cli-test/Test/Cli/Compatible/Transaction/Build.hs

```diff
@@ -120,115 +119,6 @@ hprop_compatible_shelley_create_update_proposal = propertyOnce $ H.moduleWorkspa
 
   H.diffFileVsGoldenFile outFile refOutFile
 
-hprop_compatible_shelley_transaction :: Property
-hprop_compatible_shelley_transaction = propertyOnce $ H.moduleWorkspace "tmp" $ \tempDir -> do
-  refOutFile <- H.noteTempFile tempDir "reference_tx.tx.json"
-  outFile <- H.noteTempFile tempDir "tx.tx.json"
-  let eraName = map toLower . docToString $ pretty ShelleyEra
-
-  let args =
-        [ "--fee"
-        , "5000000"
-        , "--tx-in"
-        , "596e9836a4f42661d66deb7993e4e5da310b688e85facc50fee2462e611a0c94#0"
-        , "--tx-out"
-        , "2657WMsDfac7RXyZU5nkYxPvZAh7u96FN4cp6w6581zJUR4vKUr3kofjd8MuFghFS+35999999995000000"
-        , "--update-proposal-file"
-        , inputDir <> "shelley/update-proposal.json"
-        ]
-
-  -- reference transaction
-  void . execCardanoCLI $
-    [ eraName
-    , "transaction"
-    , "build-raw"
-    ]
-      <> args
-      <> [ "--out-file"
-         , refOutFile
-         ]
-
-  -- tested compatible transaction
-  void . execCardanoCLI $
```

Comment:
Instead of deleting this test we ought to update it with the `latest` group of commands as we do care about the functionality working.

Reply:
The `--update-proposal-file` argument used in this test no longer exists in `conway`.

If we were to update this test to `latest`, it would have to do something different that is supported in `conway`.

What should it do instead?

Reply:
Ah I misread, the property.